.github/actions/setup-node/action.yml

```diff
@@ -13,13 +13,13 @@ runs:
         echo "OS_ARCH=`uname -m`" >> $GITHUB_ENV
         sudo apt -y install build-essential openssl libssl-dev pkg-config liblz4-tool
       shell: bash
-    - uses: actions/cache@v4
+    - uses: actions/cache@d4323d4df104b026a6aa633fdb11d772146be0bf
```

Comment:
There is a new tag from upstream (v4.2.3)

Reply:
Dependabot should keep action commits up to date per https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot